### ISSUE TITLE
!include config does not work

### ISSUE THREAD

**Issue by nicokant** [12rambau/pytest-copie#82]
**Describe the bug**
Trying to run the test on a project with split configuration does not work.

A simple test produces the following error:
```
 <Result expected a single document in the stream\n  in "<unicode string>", line 2, column 1:\n    !include src/includes/shared.yml\n    ^\nbut found another document\n  in "<unicode string>", line 3, column 1:\n    ---\n    ^>.exit_code
```

**To Reproduce**
at the start of your `copier.yml` add:

```
---
!include example.yml
---

```
then create a new `example.yml` with a question

Run `pytest`

test_file.py
```
def test_template_with_extra_answers(copie):
    result = copie.copy()
    assert result.exit_code == 0
```

**Comment by 12rambau** [comment]
I'm simply not aware of this mechanism from copier, can you point me to a documentation so I can understand what needs to be changed from my side ?

**Comment by nicokant** [comment]
Of course :) 
https://copier.readthedocs.io/en/stable/configuring/#include-other-yaml-files

**Comment by davidbrownell** [comment]
I believe that this can be fixed by changing the call `yaml.safe_load` to `yaml.safe_load_all` at https://github.com/12rambau/pytest-copie/blob/8c74beb10c0ca4fa1e79ae2b48793f81ff0540d7/pytest_copie/plugin.py#L74.

Additional information can be found at https://stackoverflow.com/a/14404103.

**Comment by 12rambau** [comment]
Fixed in #84
